### ISSUE TITLE
NPE in eclipse formatter

### ISSUE THREAD

**Issue by cal101** [diffplug/spotless#116]
Hi Spotlessers!

I think the following NPE in the eclipse formatter bytes me:
   https://bugs.eclipse.org/bugs/show_bug.cgi?id=500443

As far as I can see spotless is using version 4.6.1 of eclipse formatter.
Are there any plans updating to 4.6.3 soon?

I updated the eclipse formatter of my spotless installation in the past.It was not nice to do ...
Did the selection of the correct combination of jars got more easy?

Keep on going with this great project,
   Cal

**Comment by cal101** [comment]
Looks like the major work of selecting the correct jars seems to be done already in goomph.

Bumping the version strings from 4.6.1 to 4.6.3 and executing:

./gradlew -b _ext/eclipse-jdt/build.gradle publish

Gave me "version not supported"

Guess I need GOOMPH 3.8.0-SNAPSHOT.

I normally use gradle but had to do some maven lately and like it's "mvn install" that install into
local maven repo (~/.m2).
Whats the equivalent of "mvn install" that installs into "mavenLocal()" for getting spotless to use my locally built goomph?

**Comment by nedtwigg** [comment]
Thanks for bringing this to our attention!  I just published support for eclipse 4.6.3, here's how to use it:

```gradle
plugins {
  id "com.diffplug.gradle.spotless" version "3.4.0"
}

spotless {
  java {
    eclipse('4.6.3').configFile('gradle/spotless.eclipseformat.xml')
  }
}
```

We'll make `4.6.3` the default eclipse formatter in the next spotless release, or maybe `4.7.0` if it's out by then.

**Comment by cal101** [comment]
Cool, thanks!
